src/containerapp/azext_containerapp/containerapp_env_decorator.py

```diff
@@ -69,3 +75,32 @@ def get_argument_enable_workload_profiles(self):
 
     def get_argument_enable_dedicated_gpu(self):
         return self.get_param("enable_dedicated_gpu")
+
+    def get_argument_logs_dynamic_json_columns(self):
+        return self.get_param("logs_dynamic_json_columns")
+
+
+class ContainerappEnvPreviewUpdateDecorator(ContainerAppEnvUpdateDecorator):
+    def construct_payload(self):
+        super().construct_payload()
+
+        self.set_up_app_log_configuration()
```

Comment:
No need to execute `self.set_up_app_log_configuration()` here.

Reply:
We do need it because I rewrite the logic and fix the issue in core cli